src/interfaces/IERC5564Announcer.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+/// @dev Interface for calling the `ERC5564Announcer` contract, which emits an `Announcement` event
+/// to broadcast information about a transaction involving a stealth address. See
+/// [ERC-5564](https://eips.ethereum.org/EIPS/eip-5564) to learn more.
+interface IERC5564Announcer {
+  /// @dev Emitted when something is sent to a stealth address.
+  /// @param schemeId The applied stealth address scheme (such as secp25k1).
```

Comment:
nit: Should probably make it clear it's an indentifier, i.e. something like "The identifier corresponding to the applied stealth address scheme, such as 1 for secp25k1, as defined by appropriate extension EIPs"

Reply:
Updated in e204775. Note that the scheme IDs are directly defined in EIP-5564, not extension EIPs